bids/events/transform/compute.py

```diff
@@ -0,0 +1,118 @@
+'''
+Transformations that primarily involve numerical computation on columns.
+'''
+
+import numpy as np
+import pandas as pd
+from bids.events.utils import listify
+from .base import Transformation
+
+
+class scale(Transformation):
+
+    def _transform(self, data, demean=True, rescale=True):
+        if demean:
+            data -= data.mean()
+        if rescale:
+            data /= data.std()
```

Comment:
is it desired to drag NaNs out if no variance?  it just feels that this function is not exactly for zscoring/standardization, so I wondered if some "do not scale if degenerate" logic is applicable?